### ISSUE TITLE
[a11y]: PageHeader, current page breadcrumb is not identified by screen reader

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-products#5764]
### Package

@carbon/ibm-products

### Browser

Firefox

### Operating System

Windows

### Package version

`2.46.0-rc.0`

### Automated testing tool and ruleset

Manual

### Assistive technology

JAWS

### Description

Breadcrumb / Current page option / Status is not identified by screen reader

Current page option within the breadcrumb is not identified  by screen reader  (used Arrow key navigation ).

Note: Sometimes  the  current page  option got announced (In consistent ), however  "current page status " is missing .

The current page from the breadcrumb trail is not being announced by screen readers. Breadcrumbs typically provide navigation aids that help users understand their current location within a website hierarchy. However, if the current page is not announced, along with its status indication  users relying on screen readers may struggle to navigate effectively or understand their context within the site structure.

When the name, role or state of an element is not conveyed appropriately, it prevents screen reader users to interact smoothly with the page element. 

1. Ensure that the current page option  in the breadcrumb trail is  announced by screen readers.
2. also make sure the  Current page Status indication  is available  for the Screen reader users to indicate the state :
 Use `aria-current` Properties " page"  to indicate the  status.

### WCAG 2.1 Violation

4.1.2  Name Role value 

### Reproduction/example

https://ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-page-header-pageheader--with-breadcrumbs&globals=viewport:basic

### Steps to reproduce

When you scroll page header, and the page title is applied as the current page breadcrumb it needs to use `aria-current` to announce to screen reader.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems

**Comment by devadula-nandan** [comment]
we have same issue in carbon library https://github.com/carbon-design-system/carbon/issues/16749 , which could fix this.

**Comment by AlexanderMelox** [comment]
We are currently checking if this is no longer an issue and is fixed through the carbon fix. FYI @davidmenendez

**Comment by AlexanderMelox** [comment]
Problem still persists
